common/src/jni/main/cpp/NativeCrypto.cpp

```diff
@@ -4488,6 +4560,52 @@ static void NativeCrypto_asn1_write_uint64(JNIEnv* env, jclass, jlong cbbRef, jl
     }
 }
 
+static void NativeCrypto_asn1_write_null(JNIEnv* env, jclass, jlong cbbRef) {
+    CBB* cbb = reinterpret_cast<CBB*>(static_cast<uintptr_t>(cbbRef));
+    JNI_TRACE("asn1_write_null(%p)", cbb);
+
+    CBB null_holder;
+    if (!CBB_add_asn1(cbb, &null_holder, CBS_ASN1_NULL)) {
+        Errors::throwIOException(env, "Error writing ASN.1 encoding");
+        return;
+    }
+    if (!CBB_flush(cbb)) {
```

Comment:
question: why do you flush here and not while writing an oid or while writing a tag ?